.changeset/strong-ways-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": minor
+---
+
+Remove support for Node 16
```

Comment:
This would be a major release since it is a breaking change.
But I don't believe we should remove this support.
We just need to document (or have code that checks) that certain frameworks need 18.